### ISSUE TITLE
Stylesheet links in HTML emitted with local filesystem paths

### ISSUE THREAD

**Issue by jelovirt** [dita-ot/dita-ot#48]
Converted from [SourceForge issue 1328689](http://sourceforge.net/support/tracker.php?aid=1328689), submitted by cwong15

Subject: web site dita-ot.sf.net could not saved by firefox 
in local disk
Date: Friday 14 October 2005 11:44 pm
From: "I.G.W" gwijayas@gmail.com
To: cwong15@users.sourceforge.net

I don't know how to contact the webmaster at dita-ot.sf.
net, so I try
to reach by your email

There is a bug in firefox that prevent pages in
dita-ot.sourceforge.net being saved in local disk. 
example, in
http://dita-ot.sourceforge.net/doc/DITA-relnotes.html. 
Basically, the
cause of this bug is:

<link
 href="D:\eclipse\workspace\DITA-OT-1.1.
1\pmdocs\doc\xhtml\SourceForgeFile
s\doc\commonltr.css" type="text/css" rel="stylesheet" /

> <link
>  href="D:\eclipse\workspace\DITA-OT-1.1.
> 1\pmdocs\doc\xhtml\SourceForgeFile
> s\doc\dita-ot.css" type="text/css" rel="stylesheet" />

the windows path location. I have filled the bug to bugzilla 
(bugs
number 312463):

https://bugzilla.mozilla.org/show_bug.cgi?id=312463
